extensions/resteasy-reactive/quarkus-rest-client-microprofile/runtime/src/main/java/io/quarkus/rest/rest/client/microprofile/MicroProfileRestClientResponseFilter.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.rest.rest.client.microprofile;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientResponseContext;
+import javax.ws.rs.client.ClientResponseFilter;
+
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper;
+import org.jboss.resteasy.reactive.client.handlers.ClientResponseRestHandler;
+import org.jboss.resteasy.reactive.client.impl.ClientRequestContextImpl;
+import org.jboss.resteasy.reactive.client.impl.ClientResponseContextImpl;
+import org.jboss.resteasy.reactive.common.jaxrs.ResponseImpl;
+
+public class MicroProfileRestClientResponseFilter implements ClientResponseFilter {
+    private final List<ResponseExceptionMapper<?>> exceptionMappers;
+
+    public MicroProfileRestClientResponseFilter(List<ResponseExceptionMapper<?>> exceptionMappers) {
+        this.exceptionMappers = exceptionMappers;
+    }
+
+    // mstodo aint' registered!
```

Comment:
What ain't registered? You are registering this class in the Builder.

Reply:
it was failing, I didn't have to fix it at the time, added comment, fixed later and forgot to remove the comment :)

Thanks for catching it